.koppie/app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  # Connects this user object to Role-management behaviors.
+  include Hydra::RoleManagement::UserRoles
```

Comment:
does this need to be added in `lib/generators/hyrax/models_generator.rb`?

Reply:
Oh, this was removed in https://github.com/samvera/hyrax/pull/6272/files. Should it be added back in here?

Reply:
it shouldn't, i think